Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -1229,8 +1229,8 @@ void PrimaryVertexValidation::beginJob()
 
   h_runFromConfig     = EventFeatures.make<TH1I>("h_runFromConfig","run number from config;;run number (from configuration)",
 						 runControlNumbers_.size(),0.,runControlNumbers_.size());
-  for(const auto & r : runControlNumbers_){
-    h_runFromConfig->SetBinContent(r+1,runControlNumbers_[r]);
+  for(const auto r : runControlNumbers_){
```

Comment:
@davidlt this should still be `const auto&`, right?

Reply:
Technically I don't see a need for it right now, but it doesn't hurt.

Reply:
@davidlt that's true, but you have changed it and I personally didn't see a need for that

Reply:
Ah, I originally read your question regarding `const`. I changed it to `r` because there was no point in taking a reference storage-wise. I mean, we are talking here about `uint32_t` which fits perfectly fine within a register. :) It's useless here.

Reply:
alright, since I cannot imagine that the underlying type will change to something bigger, it does indeed not matter